test/acceptance/run.sh

```diff
@@ -13,21 +13,22 @@ trap clenaup 1 2 15
 function run_acceptance_test() {
     build_num=$1
     runtime=$2
+    runtimeName=$3
     echo "deploying of ${runtime} [build: ${build_num}]"
     cp -r test/acceptance/ /tmp/acceptance/
     cd /tmp/acceptance/lambda-handlers/
     npm install $INITIAL_PWD
-    serverless deploy --runtime ${runtime} --buildNumber ${build_num} || {  echo "deployment of ${runtime} [build: ${build_num}] failed" ; result=1; }
+    serverless deploy --runtime ${runtime} --runtimeName ${runtimeName} --buildNumber ${build_num} || {  echo "deployment of ${runtime} [build: ${build_num}] failed" ; result=1; }
```

Comment:
Looks like using the stage to include build number + runtime can make things easier